win32/tests/gballoc_ll_jemalloc_int/gballoc_ll_jemalloc_int.c

```diff
@@ -260,4 +263,131 @@ TEST_FUNCTION(gballoc_ll_print_stats_works)
     gballoc_ll_free(ptr);
 }
 
+static void assert_set_option_works_for_decay(const char* option_name, int64_t decay_ms, uint32_t narenas)
+{
+    char command[32];
+    int snprintf_result;
+    int64_t verify_decay_ms;
+    size_t decay_ms_size = sizeof(decay_ms);
+
+    // Arena 1 does not exist, so je_mallctl should return EFAULT
```

Comment:
>Arena 1 does not exist [](http://example.com/codeflow?start=7&length=22)

isn't this the huge arena?

Reply:
no the test deliberately skips creating this arena 1. Huge arena is the last arena(arena 4)